windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -175,7 +175,11 @@ $replacement = (Get-Content $replacement_file) -join("`r`n")
 ##############################
 #  STEP 5 - REMOVE UNNECESSARY FILES.
 ##############################
-
+# Below dll is required to put in staging_bindir to run C++ based extension
+# included gem such as winevt_c.
+$LIBSTDCPP = $RUBY_DEV_DIR + "\mingw32\bin\libstdc++-6.dll"
+$BIN = $SD_LOGGING_AGENT_DIR + "\bin"
+cp $LIBSTDCPP $BIN
```

Comment:
The `$BIN` variable doesn't buy you anything. I'd just say:
```
cp $cpp_runtime_dll $SD_LOGGING_AGENT_DIR\bin
```

Reply:
Done